### PR TITLE
[#6851] Fix loading indicator in browser

### LINKED ISSUE #6851: Fix loading indicator in browser
### User Story

As a user, I want visual feedback that a page is loading in the browser so I know to wait for it.

### Description

*Type*: Feature

*Summary*: DApps especially can be slow to load. When doing so in the Status browser, the user is faced with a static-looking, blank screen. There's no indicator that the page is still loading.

#### Expected behavior
Display a loading indicator in the browser for slow loading pages. It would be nice to follow convention with a horizontal progress bar:

#### Actual behavior
Slow loading pages appear stuck:

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #6851

### CODE REVIEW

File: src/status_im/ui/screens/browser/views.cljs [hunk 1]
```diff
@@ -103,18 +104,44 @@
                                :accessibility-label :refresh-page-button}
     [icons/icon :icons/refresh]]])
 
+(views/defview loading-indicatior [parent-width]
+  (views/letsubs [anim-width (animation/create-value (* 0.15 parent-width))
+                  anim-x (animation/create-value 0)
+                  easing-in (fn [n] {:toValue (* n parent-width)
```
Comment (jeluard) [inline src/status_im/ui/screens/browser/views.cljs:110]: Maybe those could be extracted as utility functions?

**Review by jeluard — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by asemiankevich** [comment]
@flexsurfer @errorists so we have 2 different indicators. is that right? 

1. When accessing CryptoKitties from Collectibles section:

image

2. When open ENS dapp from Profile (or just any random URL)

image

**Comment by flexsurfer** [comment]
@asemiankevich @errorists i think  it's out of the scope of this PR, collectables screens need more plove for sure

**Comment by asemiankevich** [comment]
@flexsurfer sure, agree

**Pull request merged**